src/Elmish/JsCallback.purs

```diff
@@ -63,53 +60,61 @@ jsCallback0 fn = mkJsCallback fn ignoreErrors
 -- |
 -- |       -- PureScript:
 -- |       createElement' theView_
--- |           { onSave: jsCallback0 $ Console.log "Save"
--- |           , onCancel: jsCallback0 $ Console.log "Cancel"
--- |           , onFoo: mkJsCallback
--- |                (\(bar::Int) (baz::Int) ->
--- |                     Console.log $ "bar = " <> show bar <> ", baz = " <> show baz
--- |                )
--- |                (\err -> pure unit {- ignore errors -})
+-- |           { onSave: jsCallback $ Console.log "Save"
+-- |           , onCancel: jsCallback $ Console.log "Cancel"
+-- |           , onFoo: jsCallback \(bar::String) (baz::Int) ->
+-- |               Console.log $ "bar = " <> bar <> ", baz = " <> show baz
 -- |           }
 -- |
 -- |      // JSX:
 -- |      export const TheView = props =>
 -- |        <div>
 -- |          <button onClick={props.onSave}>Save</button>
 -- |          <button onClick={props.onCancel}>Cancel</button>
--- |          <button onClick={() => props.onFoo("bar", "baz")}>Foo</button>
+-- |          <button onClick={() => props.onFoo("bar", 42)}>Foo</button>
 -- |        </div>
 -- |
 -- | In this example, the parameters `bar` and `baz` will undergo validation at
```

Comment:
`baz` was removed from the example, but still referred to here.

Reply:
`baz` hasn't been removed, it was just changed from a string to an int, which is how it's expected to be on the PureScript side: https://github.com/collegevine/purescript-elmish/pull/19/files#diff-d900e91a5a6c6d7baa84f2fc05fe8a8bR65-R66

The original example had a mistake in it.